.github/workflows/test_python_ver_matrix.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9, '3.10']
+        python-version: [3.8, 3.9, '3.10', '3.11']
```

Comment:
```suggestion
        python-version: [3.9, '3.10', '3.11']
```

Reply:
We wanted to follow https://numpy.org/neps/nep-0029-deprecation_policy.html#drop-schedule, then we should wait another 6 months.

Reply:
ah 👍